Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -250,6 +256,8 @@ private static bool IsYamlFile(string filePath)
             }
         }
 
+        private static List<string> nonClassDictionary = new List<string>();
```

Comment:
could you put this above along with the rest of dictionaries?

Reply:
will do